### ISSUE TITLE
Tracker Listing Panic

### ISSUE THREAD

**Issue by zquestz** [jhalter/mobius-hotline-client#1]
Noticed that there was some odd output in the Tracker list on a few fetches, and one time it panic'd. Trace below:

```
panic: runtime error: index out of range [10] with length 10 [recovered]
	panic: runtime error: index out of range [10] with length 10

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/application.go:291 +0x45
panic({0x6a6a40?, 0xc00001e390?})
	/usr/lib/go/src/runtime/panic.go:785 +0x132
github.com/jhalter/mobius/hotline.serverScanner({0xc0003895a0?, 0xc000389002?, 0x7541d78e45b8?}, 0xf8?)
	/home/quest/go/pkg/mod/github.com/jhalter/mobius@v0.13.0/hotline/tracker.go:175 +0xbe
bufio.(*Scanner).Scan(0xc0005bb868)
	/usr/lib/go/src/bufio/scan.go:150 +0xac
github.com/jhalter/mobius/hotline.GetListing({0x7541d741c198, 0xc000066080})
	/home/quest/go/pkg/mod/github.com/jhalter/mobius@v0.13.0/hotline/tracker.go:140 +0x385
mobius-hotline-client/ui.(*Client).Start.func3()
	/home/quest/src/mobius-hotline-client/ui/ui.go:558 +0x6a
github.com/rivo/tview.(*List).InputHandler.func1(0xc000101c08?, 0xc000294900?)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/list.go:613 +0xcf
github.com/rivo/tview.(*List).InputHandler.(*Box).WrapInputHandler.func2(0xc0002d2280?, 0x40ea85?)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/box.go:167 +0x50
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc0002d0020, 0xc000126030)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/flex.go:255 +0xd7
github.com/rivo/tview.(*Flex).InputHandler.(*Box).WrapInputHandler.func2(0xc00028e5d0?, 0x40ea85?)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/box.go:167 +0x50
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc0002d0020, 0xc000126030)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/flex.go:255 +0xd7
github.com/rivo/tview.(*Flex).InputHandler.(*Box).WrapInputHandler.func2(0xc00028e3f0?, 0xc000101c08?)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/box.go:167 +0x50
github.com/rivo/tview.(*Pages).InputHandler.func1(0xc0002d0020, 0xc000126030)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/pages.go:323 +0xa2
github.com/rivo/tview.(*Pages).InputHandler.(*Box).WrapInputHandler.func2(0xc00028e330?, 0xc0005bbb98?)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/box.go:167 +0x50
github.com/rivo/tview.(*Application).Run(0xc0002d60e0)
	/home/quest/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240625185742-b0a7293b8130/application.go:410 +0x9a2
mobius-hotline-client/ui.(*Client).Start(0xc0002d6000)
	/home/quest/src/mobius-hotline-client/ui/ui.go:605 +0x8f9
main.main()
	/home/quest/src/mobius-hotline-client/main.go:81 +0xd8f
```

**Comment by jhalter** [comment]
Thanks, I'll take a look.

**Comment by jhalter** [comment]
Fixed in [v0.1.0](https://github.com/jhalter/mobius-hotline-client/releases/tag/v0.1.0)
